templates/observability-remote-write-proxy-oidc-secret.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This secret is only used for development purposes.
For our stage and production environment different secrets will be created and configured